aunt-leahs-functions/location/index.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(context, req) {
 	connection.on('connect', (error) => {
 		if (error) {
 			context.log('Error: ', error);
-			context.done();
+			context.done(error);

```

Comment:
Maybe rename `error` to `err` to be consistent with the other files

Reply:
We should settle on a convention because I see that Justin does the opposite (change " to ') in his PR